gnocchi/service.py

```diff
@@ -85,9 +102,6 @@ def prepare_service(args=None, conf=None,
                                                 'rest', 'policy.json'))
     conf.set_default('policy_file', cfg_path, group='oslo_policy')
 
-    log.set_defaults(default_log_levels=log.get_default_log_levels() +
-                     ["passlib.utils.compat=INFO"])
```

Comment:
Yes we can. There's a function to help with that in daiquiri, though it does not seem we really want it. The logging is set by default to WARNING for everything – which ought to be enough, and INFO/WARNING/DEBUG for Gnocchi depending on the options.